src/main/java/com/asu/cse545/group12/dao/UserDaoImpl.java

```diff
@@ -30,6 +30,7 @@ public int insertRow(Users user) {
 		Transaction tx = session.beginTransaction();  
 		user.setUserStatus("InActive");
```

Comment:
above we are use 'A' here we are using 'Inactive' lets stick to acitve or inactive